cypress/support/pages/course/pageCourses.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+class Courses {
+  static #createFAB = '[name="fab-icon"]'
+  static #courseTitle = '[data-testid="coursename"]'
+  static #nextButton = '[id="nextSection"]'
+  static #nextContinueButton = '[data-submit-label="Kurs anlegen und Weiter"]'
+  static #goToCourseOverviewButton = '[data-testid="zur-uebersicht-btn"]'
+  //static #threeDotButton = '[type="button"]'
```

Comment:
I would suggest to remove the dead code i.e. unused selectors. Later when we add new scenarios than we can add them.

Reply:
updated!